src/main/resources/META-INF/spring/spring.factories

```diff
@@ -0,0 +1,14 @@
+# AutoConfiguration
```

Comment:
Do you really need all of this to make it work?

Reply:
Yes. Actually `spring boot 3.0.1` doesn't have the way to auto-config `gRPC`